db/migrate/20150102191126_add_default_avatar_url.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultAvatarUrl < ActiveRecord::Migration
+  def change
+    change_column :users, :avatar, :string, default: "https://s3-us-west-2.amazonaws.com/tipsy-tally/default.jpg"
```

Comment:
given you've done a great job here supplying a `default_url` in the uploader class, it's questionable whether this is required. I would probably lean for relying on my software to set this default, as it is probably subject to change.